src/clib/lib/config/config_schema_item.cpp

```diff
@@ -473,8 +473,8 @@ bool config_schema_item_validate_set(const config_schema_item_type *item,
                         if (!util_is_executable(value))
                             config_error_add(
                                 error_list,
-                                util_alloc_sprintf(
-                                    "Could not locate executable:%s ", value));
+                                util_alloc_sprintf("File not executable:%s ",
```

Comment:
not a big deal at all, just wondered if this is black formatting? If not - i think i like how it was better - and with your change only one line is changed.

Reply:
i think this might be `clang-format`, or more precisely `Ubuntu clang-format version 14.0.0-1ubuntu1`

I thought I was being smart and made it part of my precommit hook to avoid noise in CI, but it might be different from what's run in CI.

let me remove the formatting change and see if i get complaints : )

Reply:
it seems it is necessary after all - [CI complains](https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fequinor%2Fert%2Fpull%2F3988%23discussion_r987806104&data=05%7C01%7Cvkra%40equinor.com%7Cec2ebe076c4a40b2a08c08daa6c2c49e%7C3aa4a235b6e248d591957fcf05b459b0%7C0%7C0%7C638005652693335689%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=sAjLvLo3IaDvVtgdTNKn1zNyFA%2FlH%2FrWiKYTq157Gz8%3D&reserved=0) about the file in question once i change the formatting : /